crates/fluvio-protocol/benches/bench.rs

```diff
@@ -0,0 +1,71 @@
+#![feature(test)]
+
+extern crate test;
```

Comment:
don't think this is still needed.

Reply:
I took it from here: https://doc.rust-lang.org/unstable-book/library-features/test.html

And when I try to remove it, Rust is not able to resolve `test` crate.

```
error[E0432]: unresolved import `test`
 --> crates/fluvio-protocol/benches/bench.rs:6:9
  |
6 |     use test::{Bencher, black_box};
  |         ^^^^ use of undeclared crate or module `test`
```

As I understand this is because benchmarks still unstable internally and we have to introduce it to the scope explicitly, theres this note in the link above about it:

> Note the test feature gate, which enables this unstable feature.